### PR TITLE
Updated syntax as per ES6

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
#### Summary

Replaced var with const or let as appropriate.
#### Related issues

Resolving #16662 
#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/element/attributes/index.md [hunk 1]
```diff
@@ -49,14 +49,14 @@ with id "paragraph", and prints each attribute's value.
...
+     const paragraph = document.getElementById("paragraph");
+     const result = document.getElementById("result");
 
      // First, let's verify that the paragraph has some attributes
      if (paragraph.hasAttributes()) {
-       var attrs = paragraph.attributes;
-       var output = "";
-       for(var i = attrs.length - 1; i >= 0; i--) {
+       const attrs = paragraph.attributes;
+       const output = "";
```
Comment (wbamberg) [inline files/en-us/web/api/element/attributes/index.md:58]:
```suggestion
       let output = "";
```

...output gets reassigned at line 60.

File: files/en-us/web/api/element/closest/index.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ if (!Element.prototype.matches) {
 
 if (!Element.prototype.closest) {
   Element.prototype.closest = function(s) {
-    var el = this;
+    const el = this;
```
Comment (wbamberg) [inline files/en-us/web/api/element/closest/index.md:86]: The changes you've made in this file are in the "Polyfill" section. Please don't update polyfills in this project.

File: files/en-us/web/api/element/getattributenames/index.md [hunk 3]
```diff
@@ -79,10 +79,10 @@ for (let name of element.getAttributeNames()) {
 ```js
 if (Element.prototype.getAttributeNames == undefined) {
   Element.prototype.getAttributeNames = function () {
-    var attributes = this.attributes;
-    var length = attributes.length;
-    var result = new Array(length);
-    for (var i = 0; i < length; i++) {
+    const attributes = this.attributes;
```
Comment (wbamberg) [inline files/en-us/web/api/element/getattributenames/index.md:82]: This is also a change to the polyfill.

File: files/en-us/web/api/element/mouseenter_event/index.md [hunk 4]
```diff
@@ -94,8 +94,8 @@ Styling the `div` to make it more visible.
 #### JavaScript
 
 ```js
-var enterEventCount = 0;
-var leaveEventCount = 0;
+const enterEventCount = 0;
+const leaveEventCount = 0;
```
Comment (wbamberg) [inline files/en-us/web/api/element/mouseenter_event/index.md:98]:
```suggestion
let enterEventCount = 0;
let leaveEventCount = 0;
```

...these are reassigned at 104 and 110.

File: files/en-us/web/api/element/mouseleave_event/index.md [hunk 5]
```diff
@@ -88,8 +88,8 @@ Styling the `<div>` to make it more visible.
 #### JavaScript
 
 ```js
-var enterEventCount = 0;
-var leaveEventCount = 0;
+const enterEventCount = 0;
+const leaveEventCount = 0;
```
Comment (wbamberg) [inline files/en-us/web/api/element/mouseleave_event/index.md:92]:
```suggestion
let enterEventCount = 0;
let leaveEventCount = 0;
```

...also reassigned.

File: files/en-us/web/api/element/scrollwidth/index.md [hunk 6]
```diff
@@ -73,7 +73,7 @@ A number.
     </button>
   </body>
   <script>
-    var buttonOne = document.getElementById('aButton'),
+    const buttonOne = document.getElementById('aButton'),
     buttonTwo = document.getElementById('anotherButton'),
     divOne = document.getElementById('aDiv'),
     divTwo = document.getElementById('anotherDiv');
```
Comment (wbamberg) [inline files/en-us/web/api/element/scrollwidth/index.md:79]:
```suggestion
    const buttonOne = document.getElementById('aButton');
    const buttonTwo = document.getElementById('anotherButton');
    const divOne = document.getElementById('aDiv');
    const divTwo = document.getElementById('anotherDiv');
```

(not strictly needed, but much better to have one declaration per line.)

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thank you @PrajwalBorkar , nice work. Just a few comments.

File: files/en-us/web/api/element/closest/index.md (hunk 2 above)
Comment (PrajwalBorkar) [reply files/en-us/web/api/element/closest/index.md:86]: Ok from now onwards I will skip the Polyfill section. Should I revert this change?

File: files/en-us/web/api/element/getattributenames/index.md (hunk 3 above)
Comment (PrajwalBorkar) [reply files/en-us/web/api/element/getattributenames/index.md:82]: Should I undo this changes also?

**Review by wbamberg — APPROVED** [review]
👍 nice work @PrajwalBorkar !

**Pull request merged**